test/Streamly/Test/FileSystem/Event.hs

```diff
@@ -395,4 +396,4 @@ testDesc =
 main :: IO ()
 main = do
     hSetBuffering stdout NoBuffering
-    hspec $ sequence_ $ map driver testDesc
+    hspec $ sequence_ $ map driver testDesc
```

Comment:
You may have to change your editor settings or be careful about removing newline at the end of file.